src/icalendar/tests/conftest.py

```diff
@@ -71,3 +80,19 @@ def utc(request):
 ])
 def in_timezone(request):
     return request.param
+
+
+@pytest.fixture(params=[
```

Comment:
Looks fine but `@pytest.mark.parametrize` may be the cleaner solution: https://docs.pytest.org/en/7.3.x/how-to/parametrize.html

Reply:
I used this way because I parametrize the fixture: https://docs.pytest.org/en/7.3.x/how-to/fixtures.html#fixture-parametrize

Reply:
I understand :)